OpenRA.Mods.Common/Traits/Render/SelectionDecorations.cs

```diff
@@ -36,6 +36,12 @@ public class SelectionDecorationsInfo : ITraitInfo, ISelectionDecorationsInfo
 
 		public readonly string Image = "pips";
 
+		[Desc("Font for the text pips")]
+		public readonly string Font = "TinyBold";
```

Comment:
Please also add some way to set the text baseline relative to the pip artwork, otherwise we see:

![screen shot 2017-11-15 at 21 03 53](https://user-images.githubusercontent.com/167819/32860166-b26090ce-ca48-11e7-92ac-6b961f8d4360.png)